dashboard/test/ui/test_status.js

```diff
@@ -138,37 +143,42 @@ Test.prototype.updateView = function () {
   updateProgress && updateProgress();
 };
 
-Test.prototype.s3Key = function () {
+Test.prototype.s3Key = function() {
   const featureRegex = /features\/(.*)\.feature/i;
-  let result = featureRegex.exec(this.feature);
-  let featureName = result[1].replace('/', '_');
+  const result = featureRegex.exec(this.feature);
+  const featureName = result[1].replace(/\//g, "_");
```

Comment:
Looks like lots on linter clean up - mostly this change?

Reply:
yep! the two commits show which is which.